docs/developer/release/prepare.md

```diff
@@ -0,0 +1,42 @@
+# Prepare build environments
+
+## macOS
+
+### Initial setup
+
+- Ensure that the build machine has:
+  - Apple-trusted `Developer ID Application: Freedom of the Press Foundation (94ZZGGGJ3W)` code-signing certificates installed
+- Apple account must have:
+  - A valid application password for `notarytool` in the Keychain. You can verify
+    this by running: `xcrun notarytool history --apple-id "<email>" --keychain-profile "dz-notarytool-release-key"`. If you don't find it, you can add it to the Keychain by running
+    `xcrun notarytool store-credentials dz-notarytool-release-key --apple-id <email> --team-id <team ID>`
+    with the respective `email` and `team ID` (the latter can be obtained [here](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id))
+  - Agreed to any new terms and conditions. You can find those if you visit
```

Comment:
This currently is in the "Initial setup" section, but I believe we need to do this for each release